python/remage/cli.py

```diff
@@ -12,13 +13,61 @@
 from pathlib import Path
 
 import colorlog
+from lgdo import lh5
+from reboost.build_hit import build_hit
 
 from .find_remage import find_remage_cpp
 from .ipc import IpcResult, ipc_thread_fn
 
 log = logging.getLogger(__name__)
 
 
+def get_rebooost_config(
```

Comment:
at some point we should move the post-processing logic to a seperate file to make the code easier to understand, but for now it should be fine.

Reply:
yes this is important. can you make a post processing module @tdixon97?